### ISSUE TITLE
Make log file name configurable

### ISSUE THREAD

**Issue by davedoesdev** [theupdateframework/tuf#303]
At the moment, it's always `tuf.log` in the current working directory.

**Comment by vladimir-v-diaz** [comment]
I like this feature request.  It had occurred to me in the past, but I was fine with it being fixed during development.

Instead of having a fixed path for the log file (See https://github.com/theupdateframework/tuf/blob/develop/tuf/log.py#L77), we can make it configurable by adding an option to [conf.py](https://github.com/theupdateframework/tuf/blob/develop/tuf/conf.py) (which is where we store all configurable settings).

What about another option to disable file logging entirely?

**Comment by davedoesdev** [comment]
Yes, that would be useful too.

**Comment by vladimir-v-diaz** [comment]
Options to disable file logging entirely and configure the logger file were implemented in #304 
Let us know if this solution works for you.

**Comment by davedoesdev** [comment]
That works fine - ta

**Comment by davedoesdev** [comment]
Unfortunately, just importing `tuf.conf` (in order to set the log file) sets it to and creates `tuf.log`:

```shell
david@david-Latitude-E6440:~/dtuf$ rm tuf.log 
david@david-Latitude-E6440:~/dtuf$ python
Python 2.7.10 (default, Oct 14 2015, 16:09:02) 
[GCC 5.2.1 20151010] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import tuf.conf
>>> 
david@david-Latitude-E6440:~/dtuf$ ls
docs  htmlcov   python      requirements.txt  setup.py
dtuf  Makefile  README.md   setenv-hub.sh     test
dxf   pylintrc  README.rst  setenv-local.sh   tuf.log
```

**Comment by davedoesdev** [comment]
https://github.com/theupdateframework/tuf/pull/307

**Comment by vladimir-v-diaz** [comment]
Thanks for providing a fix for this!

**Comment by vladimir-v-diaz** [comment]
This issue was fixed in PR #307 and #308
